canary/producer-cpp/jobs/canary_seed.groovy

```diff
@@ -0,0 +1,84 @@
+import java.lang.reflect.*;
+import jenkins.model.*;
+import org.jenkinsci.plugins.scriptsecurity.scripts.*;
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.*;
+
+WORKSPACE="."
+GROOVY_SCRIPT_DIR="$WORKSPACE/canary/producer-cpp/jobs"
+DAYS_TO_KEEP_LOGS=2
+NUMBER_OF_LOGS=1
+MAX_EXECUTION_PER_NODE=1
+NAMESPACE="producer"
```

Comment:
Namespace should be changed to producercpp

Reply:
Just temporarily changed it back to producer for testing something